products/error_tracking/frontend/components/IssueFilters/CustomFilterForm.tsx

```diff
@@ -0,0 +1,154 @@
+import { useActions, useValues } from 'kea'
+
+import { IconPlus, IconTrash } from '@posthog/icons'
+import { LemonButton, LemonInput, LemonInputSelect, LemonSelect } from '@posthog/lemon-ui'
+
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { TaxonomicPopover } from 'lib/components/TaxonomicPopover/TaxonomicPopover'
+
+import { ErrorTrackingCustomFilter, ErrorTrackingCustomFilterOption } from '~/queries/schema/schema-general'
+
+import { customFilterFormLogic } from './customFilterFormLogic'
+import { customFiltersLogic } from './customFiltersLogic'
+
+interface CustomFilterFormProps {
+    filter: ErrorTrackingCustomFilter | null
+    onBack: () => void
+}
+
+export function CustomFilterForm({ filter, onBack }: CustomFilterFormProps): JSX.Element {
+    const { customFiltersLoading } = useValues(customFiltersLogic)
+    const { name, propertyPath, options, isValid, suggestions } = useValues(customFilterFormLogic({ filter }))
+    const {
+        setName,
+        setPropertyPath,
+        addOption,
+        removeOption,
+        updateOption,
+        updateOptionValue,
+        submitForm,
+        loadPropertySuggestions,
+    } = useActions(customFilterFormLogic({ filter }))
+
+    return (
+        <div className="space-y-6">
+            <div className="flex gap-4">
+                <div className="flex-1">
+                    <label className="block font-medium mb-2">Filter name</label>
```

Comment:
Decided to not use built-in kea forms because I had problems with dynamic popovers below